WAW.API/Chat/Domain/Models/Message.cs

```diff
@@ -0,0 +1,16 @@
+using WAW.API.Auth.Domain.Models;
+using WAW.API.Shared.Domain.Model;
+
+namespace WAW.API.Chat.Domain.Models;
+
+public class Message : BaseModel {
+  public string MessageContent { get; set; } = string.Empty;
+  public DateTime Date { get; set; }
+
+  //Relationships
+  public long UserId { get; set; }
+  public User User { get; set; }
+
+  public long ChatRoomId { get; set; }
+  public ChatRoom ChatRoom { get; set; }
```

Comment:
You can fix these warnings by making it nullable.

```suggestion
  public ChatRoom? ChatRoom { get; set; }
```